### PR TITLE
homework_7

### PR DESCRIPTION
**Pull request by dzhygil**

### CODE REVIEW

File: homeworks/oleg.dzhygil_dzhygil/homework_7/fibonacci.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+test('first test', () => {
+    expect(fibonacci(0)).toEqual([0]);
+});
+test('second test', () => {
+    expect(fibonacci(1)).toEqual([0, 1, 1]);
+});
+test('third test', () => {
+    expect(fibonacci(3)).toEqual([0, 1, 1, 2, 3]);
+});
```
Comment (markoffden) [inline homeworks/oleg.dzhygil_dzhygil/homework_7/fibonacci.test.js]: tests should describe what they are actually testing

Comment (markoffden) [reply homeworks/oleg.dzhygil_dzhygil/homework_7/fibonacci.test.js]: also, you will need to use loop to perform by element check

File: homeworks/oleg.dzhygil_dzhygil/homework_7/fibonacci.task.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+function fibonacci() {
+    const random = Math.floor(Math.random() * 50 + 50);
+    const array = [0, 1];
+    for (let i = 0; i < random; i++) {
```
Comment (markoffden) [inline homeworks/oleg.dzhygil_dzhygil/homework_7/fibonacci.task.js]: your loop will do lots of unneeded iterations, this is more suitable case for a `while` loop

**Comment by markoffden** [comment]
you have conflicts in this branch with `package-lock.json`

you have to merge updated `master` branch into this branch, resolve conflicts if there are ones, and update your PR

**Comment by dzhygil** [comment]
> you have conflicts in this branch with `package-lock.json`
> 
> you have to merge updated `master` branch into this branch, resolve conflicts if there are ones, and update your PR

made requested changes

**Pull request merged**